.github/workflows/build.yml

```diff
@@ -257,6 +299,8 @@ jobs:
 
           CONFIG_WIFI_NETWORK='test' CONFIG_WIFI_PASSWORD='password' laze build -DCARGO_ARGS+='--locked' --global --partition hash:${{ matrix.partition }} --select nightly --disable xtensa
 
+      - name: show sccache stats
```

Comment:
this maybe only do on self hosted, as the sccache action does that already if used.